bittensor_cli/cli.py

```diff
@@ -177,6 +179,9 @@ class Options:
     prompt = typer.Option(
         True,
         "--prompt/--no-prompt",
+        "--no/--yes",
+        "--prompt/--no_prompt",
+        "-n/-y",
```

Comment:
I don't like the `--no` and `-n`, because these aren't standard (like `-y`), it's unclear what `--no` would mean for the user (should it decline any Confirm Prompts?), and `-n` conflicts with netuid/netuids